extensions/wikia/UserRenameTool/UserRenameTool.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * A Special Page extension to rename users, runnable by staff members.
+ * Based on the RenameUser MediaWiki extension by Arnfjörð Bjarmason and Aaron Schulz.
+ *
+ * REQUIRES THE FOLLOWING EXTENSIONS TO BE ENABLED:
+ * StaffLog, TaskManager, Phalanx, SpecialPhalanx
+ */
+
+// check for dependencies
+if ( !defined( 'MEDIAWIKI' ) ) die( "This is MediaWiki extension and cannot be used standalone." );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)